### PR TITLE
Update Readme for the demo app

### PR DESCRIPTION
**Pull request by dmitryvlass**
### Description

Here is the initial version of the readme for the demo app, which gives users an installation guide and the API docs link.

### CODE REVIEW

**Comment by dmitryvlass** [comment]
@callumflack , @tnunamak , Do you have ideas of what else we can add to the [readme](https://github.com/corsali/vana-portrait-demo/blob/f9c96b0eadc8b44d4c01bfa30b7bd27eece2eabc/README.md ?

File: README.md [hunk 1]
```diff
@@ -1,34 +1,42 @@
...
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+**Step 3. Run the app.**
 
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
+```
+$ npm start
+```
 
-## Deploy on Vercel
```
Comment (tnunamak) [inline README.md:30]: Can we keep this? And modify it to give them specific instructions to deploy for free with Vercel?

https://nextjs.org/docs/deployment#managed-nextjs-with-vercel

https://vercel.com/new/clone?project-name=vana-portrait-demo&repository-name=vana-portrait-demo&repository-url=https%3A%2F%2Fgithub.com%2Fvana%2Fvana-portrait-demo

**Comment by dmitryvlass** [comment]
CC: @callumflack

**Comment by dmitryvlass** [comment]
Added a couple of screenshots explaining the forking the repository and sharing with the team members as @newtectonics suggested 🙏 
CC: @tnunamak @callumflack

**Pull request merged**